.prettierignore

```diff
@@ -1,10 +1,10 @@
-*.svg

```

Comment:
this is probably a silly question but what is different here? The changes look the same to me

Reply:
No such thing has a silly question ;P 
It's the end of line terminator that have changed: it was a CRLF, it's now a LF, as specified by the .prettierrc.js

Reply:
Gotcha! Thanks Louis!